src/ucs/sys/sys.h

```diff
@@ -13,6 +13,10 @@
 #  include "config.h"
 #endif
 
+#ifndef __USE_GNU
```

Comment:
why needed here?

Reply:
maybe we could remove signal.h from sys.h, and use it only in the relevant C files (along with USE_GNU)